lib/build/builder.js

```diff
@@ -177,8 +184,29 @@ class Builder {
     return process.platform === 'win32' ? 'NUL' : '/dev/null'
   }
 
+  testCompileArgs (additionalArgs: string = ''): Array<string> {
+    const result = ['test']
+    // use additional build args even when we compile the tests
+    if (additionalArgs && additionalArgs.length) {
+      const parsed = argparser(additionalArgs)
+      for (let i = 0; i < parsed.length; i++) {
+        if (parsed[i] === '-o') {
```

Comment:
Should we trim?

Reply:
Not necessary here, `argparser()` handles white space for us.